app/scenes/Settings/components/ImageUpload.js

```diff
@@ -28,7 +28,7 @@ class ImageUpload extends React.Component<Props> {
   @observable isUploading: boolean = false;
   @observable isCropping: boolean = false;
   @observable zoom: number = 1;
-  file: File;
+  @observable file: File;
```

Comment:
fixes a bug where the modal doesn't update to the latest image, it got stuck on the previous one